.github/workflows/maven.yaml

```diff
@@ -68,19 +68,20 @@ jobs:
     strategy:
       matrix:
         profile:
-          - {name: 'unit-tests',    javaver: 11, args: 'verify -PskipQA -DskipTests=false'}
-          - {name: 'qa-checks',     javaver: 11, args: 'verify javadoc:jar -Psec-bugs -DskipTests -Dspotbugs.timeout=3600000'}
-          - {name: 'compat',        javaver: 11, args: 'package -DskipTests -Dversion.hadoop=3.0.3 -Dversion.zookeeper=3.5.10'}
-          - {name: 'errorprone',    javaver: 11, args: 'verify -Perrorprone,skipQA'}
-          - {name: 'jdk17',         javaver: 17, args: 'verify -DskipITs'}
+          - {name: 'unit-tests',    distribution: adopt,  javaver: 11, args: 'verify -PskipQA -DskipTests=false'}
```

Comment:
To be consistent with the `javaver` param, maybe `distribution` should be called `javadist`?